### ISSUE TITLE
[ Request]Add more code comments to wknd

### ISSUE THREAD

**Issue by Rappa-Adobe** [adobe/aem-guides-wknd#298]
### Expected Behaviour

This is a suggestion to add more comments to the code for wknd.

**Comment by godanny86** [comment]
hey @Rappa-Adobe is there a specific area that needs more comments? i.e ImageList component?

**Comment by Rappa-Adobe** [comment]
@godanny86 Sorry, It is a vague answer but since wknd is used for reference comments throughout would be helpful. It might make sense to add as code is updated instead of doing it all at once.

**Comment by davidjgonzalez** [comment]
@Rappa-Adobe so you'd like to see comments added to pretty much everything explaining the key "jobs" its doing and any important considerations/decisions points, from Java code, to JavaScript, to CSS, Node definition, etc. 

As you mentioned this might be more of go-forward motion as code is updated and/or new code is added - almost like a definition-of-done for a PR .. "Is it well commented? Yes/No"

**Comment by godanny86** [comment]
yup makes sense @Rappa-Adobe. Also Sightly got rebranded to HTL! ;)

**Comment by godanny86** [comment]
we need to re-do the tutorial branches to get on the latest archetype, so might be able to sneak in a few more comments as we go. Glad that its being used!
